src/ansi-c/parser.y

```diff
@@ -3271,6 +3271,12 @@ cprover_function_contract_sequence_opt:
         | cprover_function_contract_sequence
         ;
 
+cprover_loop_contract:
+          cprover_contract_assigns_opt
+          loop_invariant_opt
+        | loop_invariant_opt
```

Comment:
Why do we need these two rules? Wouldn't the first rule suffice? `cprover_contract_assigns_opt` can expand to epsilon.

Reply:
I have now created cprover_contract_assigns and cprover_contract_assigns_opt. This resolves the issue.